TypeCobol/Compiler/Nodes/Data.cs

```diff
@@ -469,6 +471,39 @@ public override bool Equals(object obj)
             }
             return false;
         }
+
+        public override string ToString()
+        {
+            StringBuilder sb = new StringBuilder();
+            FormalizedCommentDocumentation doc = this.CodeElement().FormalizedCommentDocumentation;
+
+            int i = 0;
+
+            while (i < SelfAndChildrenLines.Count())
```

Comment:
There are lines of code that looks almost the same in branch 1031_Outline.
Our "Go to declaration" action now goes to the first line of formalized comment.

It seems we need property like GetFirstCodeToken()   meaning exclude first line of formalize comment.


I'll create an issue about that so we can discuss the behavior with formalized comment.
Can you at least put the string "*>>>" and "*<<<" in constants ?
Because the char `*` will change soon.

Reply:
See #1297